pkg/odo/cli/catalog/list/components.go

```diff
@@ -94,7 +103,16 @@ func (o *ListComponentsOptions) Run() (err error) {
 			supported, _ := catalog.SliceSupportedTags(image)
 			o.catalogList.Items[i].Spec.SupportedTags = supported
 		}
-		machineoutput.OutputSuccess(o.catalogList)
+		// machineoutput.OutputSuccess(o.catalogList)
```

Comment:
We can remove this temporary debug comment.

Reply:
removed.